wpilibj/src/athena/java/edu/wpi/first/wpilibj/DriverStation.java

```diff
@@ -203,7 +203,11 @@ private static void reportErrorImpl(boolean isError, int code, String error, boo
         haveLoc = true;
       }
     }
-    HAL.sendError(isError, code, false, error, locString, printTrace ? traceString : "", true);
+    if (printTrace) {
```

Comment:
Disagree. This also has the unfortunate side-effect of duplicating a call with a ton of parameters for not much benefit readability wise, imo.